exercises/raindrops/description.md

```diff
@@ -1,16 +1,14 @@
-Convert a number to a string, the contents of which depend on the number's factors.
+In this variant of the children's game [Fizz Buzz](https://en.wikipedia.org/wiki/Fizz_buzz) -- which has often been used as a screening exercise in technical job interviews -- your task is to convert an number into a string that contains raindrop sounds corresponding to certain potential factors. A factor is a number that evenly divides into another number, leaving no remainder. The simplest way to test if a one number is a factor of another is to use the [modulo operation](https://en.wikipedia.org/wiki/Modulo_operation).
 
-- If the number has 3 as a factor, output 'Pling'.
-- If the number has 5 as a factor, output 'Plang'.
-- If the number has 7 as a factor, output 'Plong'.
-- If the number does not have 3, 5, or 7 as a factor,
-  just pass the number's digits straight through.
+The rules of `raindrops` are quite simple; if a given number:
```

Comment:
```suggestion
The rules of `raindrops` are that if a given number:
```

I hesitate to use "simple" or "obvious" in the description of any Exercism exercise, because if someone finds themself stumbling, it can make them feel stupid (which would be unintentional, and also a shame).

Reply:
Fair enough, fixed.